Assembly-CSharp/ModHooksGlobalSettings.cs

```diff
@@ -22,6 +22,41 @@ public class ModHooksGlobalSettings
         /// </summary>
         public LogLevel LoggingLevel = LogLevel.Info;
 
+        /// <summary>
+        ///		Wheter to use colors in the log console.
+        /// </summary>
+        public bool UseLogColors;
+
+        /// <summary>
+        ///		The color to use for Fine logging when UseLogColors is enabled
+        /// </summary>
+        public string FineColor = "grey";
+
+        /// <summary>
+        ///		The color to use for Info logging when UseLogColors is enabled
+        /// </summary>
+        public string InfoColor = "cyan";
+
+        /// <summary>
+        ///		The color to use for Debug logging when UseLogColors is enabled
+        /// </summary>
+        public string DebugColor = "white";
+
+        /// <summary>
+        ///		The color to use for Warning logging when UseLogColors is enabled
+        /// </summary>
+        public string WarningColor = "yellow";
+
+        /// <summary>
+        ///		The color to use for Error logging when UseLogColors is enabled
+        /// </summary>
+        public string ErrorColor = "red";
+
+        /// <summary>
+        ///		The color to use when UseLogColors is disabled
+        /// </summary>
+        public string DefaultColor = "white";
+
```

Comment:
This would probably be better as its own object like `InGameConsoleSettings` to reduce clutter imo

Reply:
I agree. I'll do this tomorrow

Reply:
also a heads up with the doc comments here, they are using tabs instead of spaces which should probably be fixed as well

Reply:
also I think the debug info color should be the same as the default debug color because they're basically the same log level. Cyan can be quite a big color contrast to the current colors.

Reply:
Well, they idea is that every level has a different color